### ISSUE TITLE
discussion: List of must have features for the ayo.tech website

### ISSUE THREAD

**Issue by moe-dizzle** [ayojs/ayojs.tech#5]
Ayo is Humans before technology. So one of the core principles of Ayo is human accessibility. Thus while we are discussing what build/framework to use for the website (Issue #3) , I think it is also important to discuss what features this website must have to abide by Ayo core principles.

So If you have any thoughts or features that you think would great for the website, please post it and I will compile a list below.

**Potential Must-Haves List** 

- Must be beginner-friendly 

- Must be able to support language translations

- Website must be accessible with a screen reader

- font-sizes and contrast ratios should be high enough to be easily readable by low-vision users

- easily readable and navigable regardless of device (responsive, etc).

- Aim for the website to pass [WCAG 2.0 AA Standards](https://www.w3.org/WAI/WCAG20/quickref/?currentsidebar=%23col_customize&levels=aaa&techniques=failures&technologies=smil%2Cpdf%2Cflash%2Csl)

**Comment by neurodynamic** [comment]
Suggest: font-sizes and contrast ratios should be high enough to be easily readable by low-vision users.

**Comment by Kataract** [comment]
For accessibility, since the site is static and shouldn't have any interactable content beyond links or buttons, it's probably best to aim for having the site pass [WCAG 2.0 AA](https://www.w3.org/WAI/WCAG20/quickref/?currentsidebar=%23col_customize&levels=aaa&techniques=failures&technologies=smil%2Cpdf%2Cflash%2Csl).  This should be easy to accomplish with any static site generator we select based on the discussion so far in #3.

I would also like to toss in that the site should be easily readable and navigable regardless of device (responsive, etc).

**Comment by nbarnes** [comment]
Tool we might want to look into for accessibility testing: https://github.com/github/accessibilityjs
